### PR TITLE
Updating for 2014 and 2015 "winners"

### PR DESCRIPTION
**Pull request by superic**

### CODE REVIEW

**Comment by pdokas** [comment]
:+1:

**Comment by jeremyruppel** [comment]
:+1:

File: index.php [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
 ul {
 list-style: none inside;
+padding: 0;
```
Comment (pdokas) [inline index.php]: The little things are the most satisfying things.

**Comment by pfischermx** [comment]
I don't see 2015 here which was won by Kini when doing an endless loop with accounts/dbflaux. It took 2 hours for the site to recover because the deploy process would _also_ verify if DBflaux/site was up so then we were not even able to deploy. Ben needed to hack the deploy to get rid of that and I needed to hack the PHP code to revert Kini change on prod.

File: index.php [hunk 2]
```diff
@@ -50,26 +51,16 @@
...
-	<li>Feb: Tague Griffith</li>
-	<li>July: Sean Perkins</li>
-	<li>July: Peter Norby</li>
-	<li>July: Richard Bing-Wo</li>
-</ul>
--->
-
-<h2>2013 Winner</h2>
-<p><strong>Matt Jennings</strong></p>
+<h2>2015 Winner</h2>
```
Comment (hc5duke) [inline index.php]: @pfischermx here's 2015

**Comment by superic** [comment]
@pfischermx https://github.com/iamcal/grant-pattishall-award.com/pull/7/files#diff-828e0013b8f3bc1bb22b4f57172b019dR55

**Comment by pfischermx** [comment]
Oh yeah, the stupid git diff.

:+1:

**Comment by bhautikj** [comment]
:+1:

**Comment by Kaizhi** [comment]
:+1:

**Comment by iamcal** [comment]
deployed: **http://grant-pattishall-award.com/**

**Pull request merged**